### ISSUE TITLE
Alternatives to nested tables?

### ISSUE THREAD

**Issue by engslattfrida** [sebgroup/design-library#80]
I am currently helping out to design a table for Säljstöd where they can see history for Swish agreement for private and corporate customers. The team in KL wonders what we think about ensted tables or what can be a good option to this. I have thought about it but find it very complex, and based on design library I have used a table with sections. I made a quick example of this, please see picture below. I also attach the picture of what it looks like right now with the nested tables. What do we think? Any other good ideas of how this can be solved?

![tabell nested KL](https://user-images.githubusercontent.com/55842927/65750504-dcafaf80-e108-11e9-9df1-fe15e94735f2.jpg)
![tabell nested](https://user-images.githubusercontent.com/55842927/65750506-dcafaf80-e108-11e9-934a-770b6c11c492.PNG)

**Comment by ghost** [comment]
@engslattfrida The content inside the expanded row, that today also are hidden under one accordion per section, how many rows can it be? 
I think you should make a table with more real life content to see how it feels. The section header gives a good overview in this case, but what if "kontohistorik has like 20 rows? Maybe the accordion is neccessary? How do the users use this table? /Jessi

**Comment by ulde01** [comment]
ping @conandx

**Comment by conandx** [comment]
Yes - it's a bit confusing with all the expanding areas going on. 
And slightly confusing to understand the information.
Have you considered moving one part of the detail to a side-panel instead. Or maybe even the whole thing?

Perhaps we could have a sit down :)

**Comment by engslattfrida** [comment]
No I haven't, maybe that could be something to try. A sit down sounds good. I will book you!
